kusto/conn.go

```diff
@@ -192,6 +193,21 @@ func (c *Conn) doRequestImpl(
 	headers http.Header,
 	errorContext string) (http.Header, io.ReadCloser, error) {
 
+	// Replace non-ascii chars in headers with '?'
+	for _, values := range headers {
+		var builder strings.Builder
+		for i := range values {
```

Comment:
Q: How come values is a range on top of headers?

Reply:
They made it so headers can be repeated, so each header is an array of strings (if that's what you meant)

Reply:
Thanks!